src/System.Data.Common/tests/System/Data/Common/DataColumnMappingTest.cs

```diff
@@ -0,0 +1,70 @@
+using System.Data.Common;
+using Xunit;
+
+namespace System.Data.Tests.Common
+{
+    public class DataColumnMappingTest
+    {
+        [Fact]
+        public void DataSetColumn_Get_WhenNotSet()
+        {
+            DataColumnMapping dataColumnMapping = new DataColumnMapping();
+
+            string dataSetColumn = dataColumnMapping.DataSetColumn;
+
+            Assert.Equal("", dataSetColumn);
+        }
+
+        [Fact]
+        public void SourceColumn_Get_WhenNotSet()
+        {
+            DataColumnMapping dataColumnMapping = new DataColumnMapping();
+
+            string sourceColumn = dataColumnMapping.SourceColumn;
+
+            Assert.Equal("", sourceColumn);
+        }
+
+        [Fact]
+        public void GetDataColumnBySchemaAction_String_String_DataTable_Type_MissingSchemaAction_MissingDataTableThrowsException()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("dataTable", () => DataColumnMapping.GetDataColumnBySchemaAction("", "", null, typeof(string), new MissingSchemaAction()));
+        }
+
+        [Fact]
+        public void GetDataColumnBySchemaAction_String_String_DataTable_Type_MissingSchemaAction_MissingDataSetColumnReturnsNull()
+        {
+            DataColumn dataColumn = DataColumnMapping.GetDataColumnBySchemaAction("", null, new DataTable(), typeof(string), new MissingSchemaAction());
+
+            Assert.Null(dataColumn);
+        }
+
+        [Fact]
+        public void GetDataColumnBySchemaAction_String_String_DataTable_Type_MissingSchemaAction_DataColumnExpressionExistsThrowsException()
+        {
+            DataColumn priceColumn = new DataColumn();
+            priceColumn.DataType = Type.GetType("System.Decimal");
```

Comment:
why not just typeof(decimal)?

Reply:
@danmosemsft and @stephentoub Shall I implement @ViktorHofer suggestions or is this pull request okay as is?

Thanks,

Michael

Reply:
Yes please fix the GetType.
BTW @ViktorHofer is on the .NET team 😃

Reply:
Hi @danmosemsft, @stephentoub and @ViktorHofer, 

I have updated the code per Viktor's suggestions.

If you have any questions, please let me know.

Vielen Dank,

der Michael